Super_Awesome_DNA_Finder.Rmd

```diff
@@ -0,0 +1,454 @@
+	if (length(cod2[1]) == 0 | length(pos_stop2+2) == 0) {pos2 <- "NONE"} else {pos2 <- dna[cod2[1]:(pos_stop2+2)]}
+	if (length(cod3[1]) == 0 | length(pos_stop3+2) == 0) {pos3 <- "NONE"} else {pos3 <- dna[cod3[1]:(pos_stop3+2)]}
+	if (length(cod4[1]) == 0 | length(pos_stop4+2) == 0) {pos4 <- "NONE"} else {pos4 <- revcomp[cod4[1]:(pos_stop4+2)]}
+	if (length(cod5[1]) == 0 | length(pos_stop5+2) == 0) {pos5 <- "NONE"} else {pos5 <- revcomp[cod5[1]:(pos_stop5+2)]}
+	if (length(cod6[1]) == 0 | length(pos_stop6+2) == 0) {pos6 <- "NONE"} else {pos6 <- revcomp[cod6[1]:(pos_stop6+2)]}
+	
+	cat("ORF 1", file="output.txt", sep="\n")
+	cat(paste("Start:", cod1[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop1), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos1, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+
+	
+	cat("ORF 2", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod2[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop2), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos2, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+	
+	cat("ORF 3", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod3[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop3), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos3, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+
+	cat("ORF 4", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod4[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop4), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos4, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+	
+	cat("ORF 5", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod5[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop5), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos5, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+	
+	cat("ORF 6", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod6[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop6), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos6, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+
+	return(cat(longest))
+
+}
+```
+
+Here is an example of the function working on the DNA input from the "sampleDNA.csv" sheet:
+```{r}
+openreadingframe(orf=dna, pos=pos_dna)
+read.delim("output.txt")
```

Comment:
Instead of reading from a ready-made output file, your script should generate it. I tried rendering your markdown, and it doesnʻt generate the same output as I see here. It has errors.